.gitignore

```diff
@@ -8,3 +8,4 @@ erl_crash.dump
 
 # Downloaded files
 /downloads
+mix.lock
```

Comment:
Hm, I need to look into the recommendation on this. It's not part of the default .gitignore generated by Mix. Did it cause issues for you that it was in version control?